lottie/src/main/java/com/airbnb/lottie/ContentGroup.java

```diff
@@ -96,6 +99,31 @@
     lottieDrawable.invalidateSelf();
   }
 
+  @Override public String getName() {
+    return name;
+  }
+
+  @Override public void addColorFilter(@Nullable String layerName, @Nullable String contentName,
+      @Nullable ColorFilter colorFilter) {
+    for (int i = 0; i < contents.size(); i++) {
+      final Content content = contents.get(i);
+      if (content instanceof DrawingContent) {
+        final DrawingContent drawingContent = (DrawingContent) content;
+
+        if (layerName == null) {
```

Comment:
You can think of contents as children of a layer if you're not that familiar with After Effects. Given that, once you are in a DrawingContent (that doesn't extend BaseLayer), layerName isn't relevant.
CompositionLayer is the only thing that should need layerName.